### DISCUSSION TITLE
Incorrect example in syntax help page

### DISCUSSION THREAD

**Discussion by isaacl** [community/community#9937] [Other Feature Feedback, Questions, & Ideas] [answered]
In [cs.github.com/about/syntax#path](https://cs.github.com/about/syntax#path):

> To match only a specific filename (i.e. and not part of the path), you could use a regular expression:
> ```
> path:/(^|\\/)/README\.md$/
> ```

Pasting this example into code search does not work.  I believe it should be:
```
path:/(^|\/)README\.md$/
```

**Comment by colinwm** [comment, marked as answer]
Great find. This is fixed now, thanks for reporting!
